src/module.c

```diff
@@ -4302,6 +4302,8 @@ RedisModuleCallReply *RM_Call(RedisModuleCtx *ctx, const char *cmdname, const ch
         }
         acl_retval = ACLCheckAllUserCommandPerm(ctx->client->user,c->cmd,c->argv,c->argc,&acl_errpos);
         if (acl_retval != ACL_OK) {
+            sds object = (acl_retval == ACL_DENIED_CMD) ? c->cmd->name : c->argv[acl_errpos]->ptr;
+            addACLLogEntry(ctx->client, acl_retval, -1, object, ctx->client->user->name);
```

Comment:
are the two tests added for ACL LOG testing both of these? (cmd name and key name)?

Reply:
only one of them, i will add the other one as well